doc/man1/openssl-verify.pod.in

```diff
@@ -390,9 +391,9 @@ general form of the error message is:
 The first line contains the name of the certificate being verified followed by
 the subject name of the certificate. The second line contains the error number
 and the depth. The depth is number of the certificate being verified when a
-problem was detected starting with zero for the certificate being verified itself
-then 1 for the CA that signed the certificate and so on. Finally a text version
-of the error number is presented.
+problem was detected starting with zero for the certificate being the target of
+the verification itself then 1 for the CA that signed the target certificate
```

Comment:
I've re-phrased this to 
```
problem was detected starting with zero for the target ("leaf") certificate
itself then 1 for the CA that signed the target certificate and so on.
```
(not using "end-entity" here because in this document "leaf" is used instead).